src/redux/reducers/notificationReducer.ts

```diff
@@ -0,0 +1,24 @@
+import {createReducer} from '@reduxjs/toolkit';
+
+const initialState = {
+    isLoading: true,
+    error: null,
+    notifications: [],
+};
+
+export const notificationReducer = createReducer(initialState, {
+    GET_NOTIFICATIONS_REQUEST: state => {
```

Comment:
We can use the builder API. See https://redux-toolkit.js.org/api/createreducer#usage-with-the-builder-callback-notation